components/service-operator/apis/database/v1beta1/postgres_types.go

```diff
@@ -24,26 +24,44 @@ import (
 
 // AWS allows specifying configuration for the Postgres RDS instance
 type AWS struct {
-	DiskSizeGB   int    `json:"diskSizeGB,omitempty"`
```

Comment:
Why have we removed `diskSizeGB` from the allowed fields? Are we going with Aurora where disk doesn't matter?

Reply:
Yeah, the disk sizes don't apply to Aurora.